src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -63,68 +65,75 @@ class WorkflowActor private(binding: WdlBinding, actualInputs: Map[FullyQualifie
   private def notStarted: Receive = {
     LoggingReceive {
 
-      case Start(backend) =>
+      case Start(system, backend) =>
         _primeMover = Option(sender())
-        maybeExecutionStatusStore = Option(new ExecutionStatusStore(binding))
+        maybeExecutionStore = Option(new ExecutionStore(binding))
         maybeSymbolStore = Option(new SymbolStore(binding, actualInputs))
         maybeBackend = Option(backend)
-        executionStatusStore.runnableCalls.foreach { call =>
-          startCallActor(symbolStore, call)
-        }
-        primeMover ! Started
+        // Poll for runnable calls.
+        system.scheduler.schedule(
+          initialDelay = 0 milliseconds,
+          interval = 5 seconds,
+          receiver = self,
+          message = CheckExecutionStatus)(system.dispatcher)
+
         context.become(started)
+        primeMover ! Started
+
 
-      case bad @ _ =>
+      case bad@_ =>
         sender ! InvalidOperation(s"Received message $bad before any Start message")
     }
   }
 
+  private def createRequiredInputs(call: Call): Unit = {
```

Comment:
So for 3-step workflow...

```
outputFqn = "three_step.ps.procs"
inputFqn = "three_step.cgrep.in_file"
```

And then you copy the value from `three_step.ps.procs` to `three_step.cgrep.in_file`.

Hmm.  I can see how this is a bit janky.  I have some ideas regarding this because I ended up having to implement this two different ways in the Python version.  What I ended up settling on was having the two entries in the symbol table be like this:

|FQN|Name|Value|Type|
|---|---|---|---|
|three_step.ps|procs|/path/to/file|File|
|three_step.cgrep|in_file|**%expr:ps.procs**|File|

Whereas in the reference implementation I literally stored the string `%expr:ps.procs`, this could just as easily be some kind of Class and we figure out how to persist it later.

If somebody requested the value of `three_step.cgrep.in_file`, I'd then evaluate `%expr:ps.procs` and return that value.  This way, people can define calls like this:

```
call foobar {
    input: x=upstream.out * 100, y=upstream.out2[0]
}

And the symbol table would have two value entries for x and y that looked like `%expr:upstream.out*100` and `%expr:upstream.out2[0]`